docs/index.rst

```diff
@@ -25,69 +29,138 @@ Scale set filters are added in the same way as any other filters.
 
-      +------------+----------------------------------------------------------------------------------------+
-      | Name       | Description                                                                            |
-      +============+========================================================================================+
-      | asset      | A regular expression to match against the asset name. The scale factor is only         |
-      |            | applied to assets whose name matches this regular expression.                          |
-      +------------+----------------------------------------------------------------------------------------+
-      | datapoint  | A regular expression to match against the data point name within a matching asset.     |
-      |            | The scale factor is only applied to assets whose name matches this regular expression. |
-      +------------+----------------------------------------------------------------------------------------+
-      | scale      | The scale factor to apply to the numeric data.                                         |
-      +------------+----------------------------------------------------------------------------------------+
-      | offset     | The offset to add to the matching numeric data.                                        |
-      +------------+----------------------------------------------------------------------------------------+
+    +--------------+
+    | |scaleset_2| |
+    +--------------+
 
-  - Enable the scale-set filter and click on *Done* to activate your plugin
+  - This allows the entry of a set of asset names, datapoint names, scale factors and offsets.
 
-Example
--------
+  - list that defines a set of factors and offsets to apply particular assets and datapoints within these assets. A regular expression that is matched against the asset name and another that matches the data point name within the asset to determine if a particular scale factor and offset is applied to the datapoint value.
 
-In the following example we have an asset whose name is *environment* which contains two data points; *temperature* and *humidity*. We wish to allow two different scale factors and offsets to these two data points whilst not affecting assets of any other name in the data stream. We can accomplish this by using the following JSON document in the plugin configuration;
+.. list-table::
+   :header-rows: 1
 
-.. code-block:: JSON
+   * - Name
+     - Description
+   * - asset
+     - A regular expression to match against the asset name. The scale factor is only applied to assets whose name matches this regular expression.
+   * - datapoint
+     - A regular expression to match against the data point name within a matching asset. The scale factor is only applied to assets whose name matches this regular expression.
+   * - scale
+     - The scale factor to apply to the numeric data.
+   * - offset
+     - The offset to add to the matching numeric data.
```

Comment:
The table should be aligned with the above bullet points, so it needs to be indented to the right.